android/src/main/java/io/github/droidkaigi/feeder/notification/KaigiMessagingService.kt

```diff
@@ -0,0 +1,22 @@
+package io.github.droidkaigi.feeder.notification
+
+import com.google.firebase.messaging.FirebaseMessagingService
+import com.google.firebase.messaging.RemoteMessage
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class KaigiMessagingService : FirebaseMessagingService() {
```

Comment:
I haven't used Kaigi ~ so far, so I'd like to change the name to AppMessagingService later, but you don't need to change it with this PR. 👀